src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  return (num1) => {
+    counter++;
+
+    return (num2) => {
+      if (counter <= 3) {
+        return num1 + num2;
+      } else {
+        return counter % 2 !== 0 ? num1 + num2 : `Bzzz... Error!`;
+      }
```

Comment:
`else` is redundant here.
```suggestion
      }
      
      return counter % 2 !== 0 ? num1 + num2 : `Bzzz... Error!`;
```